Library/Formula/rcssserver.rb

```diff
@@ -75,8 +74,7 @@ def install
   end
 
   test do
-    system "#{bin}/rcssserver help | head -1 | grep 'rcssserver-#{version}'"
-    system "#{bin}/rcsslogplayer --version | tail -1 | grep 'rcsslogplayer Version'"
-    system "#{bin}/rcssmonitor --version | tail -1 | grep 'rcssmonitor Version'"
+    assert_match version.to_s,
+                 shell_output("#{bin}/rcssserver help", 1)
```

Comment:
Please don't break these assertions across multiple lines unless it is super long or something.